tools/gdb/gdb_scripts/hazard_pointers.py

```diff
@@ -0,0 +1,80 @@
+# Helper scripts for hazard pointers that can be called in gdb.
+# Example: 
+#     (in gdb) 
+#     source hazard_pointers.py
+#     dump_hazard_pointers`
+
+import gdb
+
+SESSION_IMPL_PTR = gdb.lookup_type("WT_SESSION_IMPL").pointer()
+CONN_IMPL_PTR = gdb.lookup_type("WT_CONNECTION_IMPL").pointer()
+
+# Return a list of sessions and which thread they are in
```

Comment:
Full stops at end of comments.

Reply:
Updated in [85bdb20](https://github.com/wiredtiger/wiredtiger/pull/9801/commits/85bdb20849cbd8afde6506bf1d480f568de9340e)